db/people.yml

```diff
@@ -2098,7 +2098,7 @@ mwinitch:
       begin_month: 6
       end_year: 2020
       end_month: 8
-      organization: columbiau
+      organization: Columbia University
```

Comment:
this is not a spelling mistake.  columbiau is an id that gets dereferenced in institutions.yml

Reply:
Do I have to change it back to the original in my branch?